code/modules/ai/ai_node.dm

```diff
@@ -6,18 +6,23 @@
 	icon = 'icons/effects/landmarks_static.dmi'
 	icon_state = "x6" //Pure white 'X' with black borders
 	anchored = TRUE //No pulling those nodes yo
-	invisibility = INVISIBILITY_OBSERVER //Visible towards ghosts
+	#ifdef TESTING
+	invisibility = 0
+	#else
+	invisibility = INVISIBILITY_ABSTRACT
```

Comment:
i have to put hundreds of nodes, it's ugly. Since admins will no longer need to put nodes, they don't need to see it